pkg/controllers/managementuser/rbac/namespace_handler.go

```diff
@@ -134,6 +159,27 @@ func (n *nsLifecycle) assignToInitialProject(ns *v1.Namespace) error {
 	return nil
 }
 
+func (n *nsLifecycle) GetSystemProjectName() (string, error) {
+	projects, err := n.m.projectLister.List(n.m.clusterName, initialProjectToLabels[projectpkg.System].AsSelector())
+	if err != nil {
+		return "", err
+	}
+	if len(projects) == 0 {
+		return "", nil
+	}
+	if len(projects) > 1 {
+		return "", fmt.Errorf("cluster [%s] contains more than 1 [%s] project", n.m.clusterName, projectpkg.System)
+	}
+	if projects[0] == nil {
+		return "", nil
+	}
+	return projects[0].Name, nil
+}
+
+func IsFleetNamespace(ns *v1.Namespace) bool {
+	return ns.Name == "fleet-local" || ns.Name == "fleet-default" || ns.Name == "cattle-fleet-clusters-system" || ns.Labels["fleet.cattle.io/managed"] == "true"
```

Comment:
This needs to be `fleet-clusters-system`. In 2.5 we haven't change the fleet system namespace from `fleet-system` to `cattle-fleet-system`. (https://github.com/rancher/fleet/issues/395)

Reply:
Updated.